From 6b88d71928def928f244f799188e55b07c10ba5c Mon Sep 17 00:00:00 2001
From: Lucian Poston <lucian.poston@gmail.com>
Date: Sat, 24 Oct 2015 00:51:10 -0700
Subject: [PATCH] Always link against gstreamer 1.0

Signed-off-by: Lucian Poston <lucian.poston@gmail.com>
---
 configure.ac | 13 +------------
 1 file changed, 1 insertion(+), 12 deletions(-)

diff --git a/configure.ac b/configure.ac
index eaffedf..830a9fd 100644
--- a/configure.ac
+++ b/configure.ac
@@ -884,20 +884,9 @@ config_gstreamer=no
 if test "x$enable_gstreamer" != "xno";
 then
     PKG_CHECK_MODULES([GSTREAMER],
-                      [gstreamer-0.10],
-                      [config_gstreamer=yes
-                       AC_DEFINE(HAVE_GSTREAMER, 1, [Use GStreamer for playing sounds])], [ ])
-		     
-    if test "x$config_gstreamer" != "xyes";
-    then
-        PKG_CHECK_MODULES([GSTREAMER],
                       [gstreamer-1.0],
                       [config_gstreamer=yes
-                       AC_DEFINE(HAVE_GSTREAMER, 1, [Use GStreamer for playing sounds])],
-                         [if test "x$enable_gstreamer" = "xyes"; then
-                           AC_MSG_ERROR([GStreamer development headers not found.])
-                       fi])
-    fi
+                       AC_DEFINE(HAVE_GSTREAMER, 1, [Use GStreamer for playing sounds])], [ ])
 fi
 
 
-- 
2.4.10

