diff --git a/src/mudlet-lua/lua/LuaGlobal.lua b/src/mudlet-lua/lua/LuaGlobal.lua
index 367adca..9643e9f 100644
--- a/src/mudlet-lua/lua/LuaGlobal.lua
+++ b/src/mudlet-lua/lua/LuaGlobal.lua
@@ -130,6 +130,8 @@ local packages = {
 local prefixes = {"../src/mudlet-lua/lua/", "../Resources/mudlet-lua/lua/",
     "mudlet.app/Contents/Resources/mudlet-lua/lua/", "mudlet-lua/lua"}
 
+table.insert(prefixes, "/usr/share/mudlet/lua/")
+
 local prefix
 for i = 1, #prefixes do
     if lfs.attributes(prefixes[i]) then
diff --git a/src/TLuaInterpreter.cpp b/src/TLuaInterpreter.cpp
index 261ae4c..8d7c5b6 100644
--- a/src/TLuaInterpreter.cpp
+++ b/src/TLuaInterpreter.cpp
@@ -11755,7 +11755,7 @@ void TLuaInterpreter::loadGlobal()
 #if defined(Q_OS_MAC)
     QString path = QCoreApplication::applicationDirPath() + "/../Resources/mudlet-lua/lua/LuaGlobal.lua";
 #else
-    QString path = "../src/mudlet-lua/lua/LuaGlobal.lua";
+    QString path = "/usr/share/mudlet/lua/LuaGlobal.lua";
     // Additional "../src/" allows location of lua code when object code is in a
     // directory alongside src directory as occurs using Qt Creator "Shadow Builds"
 #endif
